docs/whats-new.md

```diff
@@ -43,6 +43,10 @@ Target Release Date: Q1 2021. (Alpha releases are available).
 
 - New table category loader for Excel spreadsheets in both binary `.xls`, `.xlsb` and XML-based `.xlsx` formats.
 
+**@loaders.gl/mvt**
+
+- Binary output is now 2-3X faster for large datasets thanks to parsing directly from PBF to binary, rather than going through geoJSON as an intermediate representation
```

Comment:
Nit
```suggestion
- Binary output is now 2-3X faster for large datasets thanks to parsing directly from PBF to binary, rather than going through GeoJSON as an intermediate representation
```

Reply:
Would be great to include a table like this to explain the performance improvement:

https://github.com/visgl/deck.gl/blob/master/docs/whats-new.md#binary-mode